TTHAnalysis/python/plotter/susy-ewkino/3l/cuts_ewkino.txt

```diff
@@ -1,7 +1,12 @@
 alwaystrue: 1
-event     : categ > 0 
-inSR      : SR > 0
-haspair   : isTight || isFake
-trigger   : Triggers_ee || Triggers_mm || Triggers_em
+blinding  : !isData || run <= 274240
+3l        : nLepSel >= 3
+sanity    : is_3l
+isTight   : isTight(3, LepSel_isTight[0], LepSel_isTight[1], LepSel_isTight[2]) 
```

Comment:
Calling a cut with the same name of a function, and then also replacing the same name in fakerate files, is very bad practice. Please rename things to different names.